### ISSUE TITLE
Improve support added for importing Kibana dashboards from GitHub

### ISSUE THREAD

**Issue by kekesimarti** [filetrust/cdr-plugin-folder-to-folder#250]

**Comment by kekesimarti** [comment]
@MaNe2020 please add as much details as possible here.

**Comment by MaNe2020** [comment]
@ggrig @samarthdd 
At the moment Kibana dashboards are loaded from https://github.com/filetrust/cdr-plugin-folder-to-folder/blob/main/cdr_plugin_folder_to_folder/metadata/Metadata_Elastic.py
within
` reload_kibana_dashboards` by directly adding dashboard njsons.
These jsons are located at: https://github.com/filetrust/cdr-plugin-folder-to-folder-test-data/tree/main/kibana-dashboards

If new Dashboard is to be added (or existing one is modified) it needs to be uploaded to above repo and name should be added in Metadata_Elastic.py

We should see if there is better, more dynamic way of doing this and does it need to be part of the code or it can be done as part of docker compose.

**Comment by MaNe2020** [comment]
This also implies if do any changes to the dashboards, and VM gets shut down, those changes are lost.
1. Open dashboard
2. Add some new lences
3. Save new dashboard
4. Shut down VM
5. Start VM
6. Go back to dashboards

After step 6, changes from step 2 are lost. 

Not sure if this is to be separate issue, but it is smth to think about.
